Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -157,6 +157,7 @@ + (NSString *)bundleIdForPath:(NSString *)path {
     NSString *bundleId = [dic objectForKey:(NSString *)kCFBundleIdentifierKey];
     if (!bundleId) {
         [BPUtils printInfo:ERROR withString:@"Could not extract bundleID: %@", dic];
+        exit(1);
```

Comment:
Won't this leave a mess? e.g. Simulators that are booted/cloned but not cleaned up? Would it be better to do error checking on the callers?

Reply:
+1..

Reply:
replaced with cloneSimulator flag to turn on/off clone simulator feature